Reply/app/src/main/java/com/materialstudies/reply/ui/home/EmailViewHolder.kt

```diff
@@ -57,9 +57,8 @@ class EmailViewHolder(
         binding.root.isActivated = email.isStarred
         starImageView.setOnClickListener { view: View ->
             email.isStarred = !email.isStarred
```

Comment:
I know this doesn't fully belong to this PR, but can you change this line to:

`binding.listener?.onEmailStarChanged(email, !email.isStarred)`
?

Or create a private method in this class like

```
private fun updateEmailStarred(starred: Boolean) {
    val email = binding.email ?: return
    binding.listener?.onEmailStarChanged(email, starred)
    email.notifyChange()
}
```

And then call this method from the starImageView click listener and the `onRebounded` method?